http/src/main/resources/reference.conf

```diff
@@ -73,7 +73,7 @@ dataproc {
   customDataprocImage = "projects/broad-dsp-gcr-public/global/images/leo-dataproc-image-2-2-52-debian12-2025-05-06-13-58-25"
   # AN-503: Delete once AOU has switched to using Dataproc 2.2.X in prod
   # Cached dataproc 2.1.x image used by AOU
-  legacyAouCustomDataprocImage = "projects/broad-dsp-gcr-public/global/images/leo-dataproc-image-2-1-11-debian11-2025-05-06-16-08-29"
+  legacyAouCustomDataprocImage = "projects/broad-dsp-gcr-public/global/images/leo-dataproc-image-2-1-11-debian11-2025-05-07-16-35-36"
```

Comment:
This is now 28GB instead of almost 50GB and should avoid the creation timeout that you were seeing

Reply:
omg